resources/puppetlabs/lein-ezbake/template/pe/ext/redhat/init.suse.erb

```diff
@@ -149,27 +118,23 @@ sl_status_q() {
 }
 
 sl_status() {
+    echo -n $"Checking for service ${prog}: "
     checkproc -p "${PIDFILE}" -- "${JAVA_BIN}" "-XX:OnOutOfMemoryError=\"kill -9 %p\"" "${JAVA_ARGS}"
     rc_status -v
 }
 
 reload() {
-    echo -n $"Reloading ${prog}:"
-    /opt/puppetlabs/server/apps/${realname}/bin/${realname} reload
+    echo -n $"Reloading ${prog}: "
+    "${INSTALL_DIR}/bin/${realname}" reload
     rc_status -v
     retval=$?
 
-    if [ "$retval" = 0 ]; then
-        log_success_msg $"${prog} reloaded"
```

Comment:
Same rationale for the seemingly redundant status message line here as with the `start` command.